commons-test/src/main/java/org/infinispan/commons/test/skip/SkipUtil.java

```diff
@@ -0,0 +1,14 @@
+package org.infinispan.commons.test.skip;
+
+class SkipUtil {
+   static OS getOs() {
```

Comment:
We could remove this class and move the getOs method inside OS enum so ```SkipUtil.getOs()``` becomes ```OS.get()```.